pkg/exchange/coinbase/stream_handlers.go

```diff
@@ -25,6 +26,20 @@ const (
 	balanceChannel     types.Channel = "balance"
 )
 
+// minInterval: for the kline updater
+var minInterval = types.Interval("1s")
```

Comment:
use const

Reply:
In the `init()`, it will be updated according to supported intervals on Coinbase.
In this way, if there is any updates to the supported intervals on Coinbase, we won't need to change the min interval here.
It will be updated accordingly.